src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
Why is this necessary?

Reply:
This was automatically added when I created the tsconfig. Basically it brings types declarations to react. I have created also one folder called types for those libs that have no type support like @h1ve-ui, in types/index.d.ts (it's not completed since I only need to add it for a couple of tsx files, but you get the idea). I also created one for styled components css prop in typescript.